### ISSUE TITLE
Mindergas uploading not working

### ISSUE THREAD

**Issue by bartn** [xirixiz/dsmr-reader-docker#37]
I've enabled the mindergas upload in the DSMR config. However no data is being sent over. I have a feeling the dsmr-container cannot reach the 'outside internet'. I am pretty new to docker and I am running your repo using the docker-compose file on my Synology NAS. Not sure if I need to change the network-bridge, so it allows connecting to the internet?
note: if you believe my question doesn't relate to your repo, my apologies.

**Comment by xirixiz** [comment]
Hi Bart,

It's alright asking this question, no worries :). I recently swtiches from a self build server to a Synology nas. As I have plenty of experience with Docker, I had the same issue with all my Docker containers not be able to lookup hostnames on the internet. May I ask how you configured the network settings of your Synology? 

I configured the Synology itself as a DNS server (127.0.0.1 and f.e. 192.168.0.10) since PI-Hole was running on the Synolgy in Docker with a forwarder to the cloudfare DNS servers as forwarders. That doesn't work, I really had resolving issues within Docker containers. I have more network related issues with Docker on Synology, so if you can sent me some more info regarding your setup I might be able to help you out.

**Comment by bartn** [comment]
Hi @xirixiz , my network-config is very straight forward. Synology is connect out of the box to my router (Fritzbox) and has a static IP. DHCP and DNS are managed by the Fritzbox.

**Comment by xirixiz** [comment]
Hi Bart, totally forgot about your message (email). I`ll reply to you Tonight...

**Comment by bartn** [comment]
@xirixiz , don't worry. I found the issue yesterday. I saw that Synology DSM also couldn't fetch DSM-updates (packages update were received though). I changed the DNS settings of DSM to the standard Google DNS. This resolved the DSM-updates. And this morning I saw the Mindergas-upload also worked overnight.
This whole DNS thing might be caused by experimenting with Pi-hole and the router-settings though...
